### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by schoeberl** [mortbopet/Ripes#44]
You can read binary files, but save sould also be nice. That could help to use Ripes as the golden model for their own ISA simulation implementation.

**Comment by mortbopet** [comment]
This should already be possible. When saving in Ripes, up to three different files will be saved, as specified by the user:
- Source file
- Disassembled file
- Binary file (saved as text or as a flat binary file).
Then, once the user selects "Save As" or "Save Files As", all of the selected files will be stored, with each filename suffixed by the file type.

![binary](https://user-images.githubusercontent.com/16338943/69726392-c4680b00-1120-11ea-80aa-6b5b062f830f.png)

**Comment by mortbopet** [comment]
Indeed! Apart from a couple of new features, there's been quite a few bug fixes since then.
